tests/legacy-v1-client.yml

```diff
@@ -6,3 +6,6 @@ services:
     #
     mender-client:
         image: mendersoftware/mender-client-qemu:1.7.0
+        volumes:
+          # Provision demo certificate
+          - ./cert/cert.crt:/mnt/config/server.crt
```

Comment:
does this not also need the `      - ./extra/integration-testing/setup-mender-configuration.py:/setup-mender-configuration.py` mount?
`

Reply:
This version of the client does not use `mender-connect` or any other services that needs the certificate, so the default entrypoint script is enough to update `/etc/mender/mender.conf` to point to this mount.